change/@splunk-otel-ba9ac324-9f97-419b-890e-aa3e6b8bd70a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
if OTel was bumped by `minor`, then shouldn't we bump by minor too?

Reply:
Assuming that you reviewed changes to the API and the SDK and concluded that the minor bump is not needed - then no!